types/express-serve-static-core/index.d.ts

```diff
@@ -283,21 +284,25 @@ export interface Request extends http.IncomingMessage, Express.Request {
     acceptsLanguages(...lang: string[]): string | false;
 
     /**
-     * Parse Range header field,
-     * capping to the given `size`.
+     * Parse Range header field, capping to the given `size`.
      *
-     * Unspecified ranges such as "0-" require
-     * knowledge of your resource length. In
-     * the case of a byte range this is of course
-     * the total number of bytes. If the Range
-     * header field is not given `null` is returned,
-     * `-1` when unsatisfiable, `-2` when syntactically invalid.
+     * Unspecified ranges such as "0-" require knowledge of your resource length. In
+     * the case of a byte range this is of course the total number of bytes. If the
+     * Range header field is not given `undefined` is returned, `-1` when unsatisfiable,
+     * and `-2` when syntactically invalid.
+     *
+     * When ranges are returned, the array has a "type" property which is the type of
```

Comment:
I think this should be moved to `ByteRange` and `RequestRanges` interfaces docs

Reply:
These lines are original code document in the [Express source code](https://github.com/expressjs/express/blob/3ed5090ca91f6a387e66370d57ead94d886275e1/lib/request.js#L188-L203). Previously, this part also copied from the original code document to guide how it can be used. I put it in because it seems to me, the in-code documents for many functions in `types/express-servce-static-core/index.d.ts` are from original in-code documents of those functions. I tried to keep consistency on how we put the in-code documents.

If you insists, I can remove the comments.

Reply:
I know that they are from the source code, I've copied it 😉 
But I think that it's better when we don't put 2 pages descriptions in JSDoc as the description of the shape of return type is known by TS type and should be placed where the return type is defined.

But as the rest of the definitions do the same, we can stick to that to keep consistency.

Reply:
Ok, rogers that. I've moved partial of docs into `range-parser/index.d.ts` instead. Except the `undefined`, we still need it here, because range-parser never return `undefined`. This part belong to req.range()

Thanks for the feedback :)